pkg/users/account-details.js

```diff
@@ -164,30 +189,14 @@
                 .catch(show_unexpected_error);
     }
 
-    if (!accounts.length) {
+    if (isLoading) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-7fd6cddb-20250813-150050-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)